pkgs/tools/misc/sd-mux-ctrl/default.nix

```diff
@@ -10,19 +10,21 @@ stdenv.mkDerivation rec {
     sha256 = "0fxl8m1zkkyxkc2zi8930m0njfgnd04a22acny6vljnzag2shjvg";
   };
 
-  nativeBuildInputs = [ cmake pkg-config ];
+  nativeBuildInputs = [ cmake pkg-config installShellFiles ];
 
   buildInputs = [ libftdi1 popt ];
 
   postInstall = ''
     install -D -m 644 ../doc/man/sd-mux-ctrl.1 $out/share/man/man1/sd-mux-ctrl.1
+    installShellCompletion --cmd sd-mux-ctrl \
+      --bash ../etc/bash_completion.d/sd-mux-ctrl
   '';
 
   meta = with lib; {
     description = "Tool for controlling multiple sd-mux devices";
     homepage = "https://wiki.tizen.org/SD_MUX";
```

Comment:
Unable to access this link now

Reply:
It loads for me, just very slowly.

Reply:
I can't even resolve that domain, not sure

```
=> dig wiki.tizen.org

; <<>> DiG 9.18.27 <<>> wiki.tizen.org
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 47343
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 65494
;; QUESTION SECTION:
;wiki.tizen.org.                        IN      A

;; Query time: 2 msec
;; SERVER: 127.0.0.53#53(127.0.0.53) (UDP)
;; WHEN: Thu Aug 08 10:50:28 CST 2024
;; MSG SIZE  rcvd: 43
```